wear/src/main/java/chat/rocket/android/util/Ui.kt

```diff
@@ -0,0 +1,47 @@
+package chat.rocket.android.util
+
+import android.app.Activity
+import android.app.Fragment
+import android.widget.Toast
+import androidx.annotation.StringRes
+
+fun Activity.showToast(@StringRes resource: Int, duration: Int = Toast.LENGTH_SHORT) =
```

Comment:
Duplicated. Please remove. You can put this class in the `util` module and use it from there on the `wear` and `app` module as well.